src/test/extensions/resolveSource.spec.ts

```diff
@@ -66,25 +64,24 @@ describe("checkForUpdateWarnings", () => {
         "After updating, you must switch your Cloud Scheduler jobs to <b>PubSub</b>, otherwise your extension will stop running.",
     };
     expect(confirmUpdateWarningSpy).to.have.been.calledWith(expectedUpdateWarning);
-  });
+  }).timeout(5000);
```

Comment:
Why is this timeout needed? I've found that if timeouts are necessary, it's because some network call isn't being stubbed/mocked/nock'd correctly. I've recently forced all tests to have auth (for API calls) mocked out, so timeouts like this shouldn't be necessary. Is it possible to remove this timeout? (you can run this specific test by using `it.only` or passing the file name directly to `mocha`)